adapters/globalsun/globalsun.go

```diff
@@ -0,0 +1,148 @@
+package globalsun
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type reqBodyExt struct {
+	GlobalsunBidderExt reqBodyExtBidder `json:"bidder"`
+}
+
+type reqBodyExtBidder struct {
+	Type        string `json:"type"`
+	PlacementID string `json:"placementId,omitempty"`
+	EndpointID  string `json:"endpointId,omitempty"`
+}
```

Comment:
Same thing here. All fields to reqBodyExtBidder struct doesn't have to be exported. 
```
type reqBodyExtBidder struct {
	type        string `json:"type"`
	placementID string `json:"placementId,omitempty"`
	endpointID  string `json:"endpointId,omitempty"`
}
```